### PR TITLE
Update getting started material

### PR DESCRIPTION
**Pull request by katejeffreys**

### CODE REVIEW

File: app/2.0/start/samples/editable-name-tag/editable-name-tag.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      <input is="iron-input" placeholder="Your name here...">
+    </iron-input>
+  </template>
+
+  <script>
+    class EditableNameTag extends Polymer.Element {
+      static get is() { return "editable-name-tag"; }
+      
+      // configure the owner property
+      static get config() {
```
Comment (arthurevans) [inline app/2.0/start/samples/editable-name-tag/editable-name-tag.html]: This should be:

```
static get properties() {
  return {
    owner: {
      type: String,
      value: 'Daniel'
    }
  }
}
```

Comment (arthurevans) [inline app/2.0/start/samples/editable-name-tag/editable-name-tag.html]: Same change for configurable-name-tag sample (also, in that sample, the {{owner}} text binding should be changed to [[owner]]

File: app/2.0/start/quick-tour.md [hunk 2]
```diff
@@ -134,10 +134,10 @@ It also shows configuring the owner property from markup in `index.html`.
 
 <demo-tabs selected="0" src="http://plnkr.co/edit/DhDSeqNrmLflcQ8UZI1R?p=preview">
```
Comment (arthurevans) [inline app/2.0/start/quick-tour.md]: This plunker link goes to a 1.x sample.

**Comment by arthurevans** [comment]
We still need to:

- Update the two samples using the old syntax (configurable-name-tag and editable-name-tag), and change "local DOM" to "shadow DOM" throughout. 
- Fix the CTAs at the bottom of the doc to link to appropriate places.
- Also (sorry, missed this first time)--in the first example, please change the `ready()` callback to a `constructor()` like the `name-tag` example.

**Comment by katejeffreys** [comment]
Changes made. Can't test because can't build docs site as discussed

**Comment by katejeffreys** [comment]
@arthurevans built and tested. Ready for someone else to check it!

**Pull request merged**